src/main/fc/fc_msp.c

```diff
@@ -115,6 +115,9 @@
 #define STATIC_ASSERT(condition, name) \
     typedef char assert_failed_ ## name [(condition) ? 1 : -1 ] __attribute__((unused))
 
+#define BYTES_REMAINING_GTE(num) \
```

Comment:
Either pass `src` as argument or define this macro just before/inside the function and `#undef` if after it

`BYTES_AVAILABLE` may be better name